### PR TITLE
Add alt names

### PR DESCRIPTION
**Pull request by HighwayofLife**
Adding support for additional DNS names and IPs (Alt names) used for PKI (certificates) in rke.

### CODE REVIEW

File: apis/management.cattle.io/v3/rke_types.go [hunk 1]
```diff
@@ -237,6 +237,8 @@ type AuthnConfig struct {
 	Strategy string `yaml:"strategy" json:"strategy,omitempty"`
 	// Authentication options
 	Options map[string]string `yaml:"options" json:"options,omitempty"`
+	// List of additional hostnames and IPs to include in the api server PKI cert
+	Sans []string `yaml:"sans" json:"sans,omitempty"`
```
Comment (ibuildthecloud) [inline apis/management.cattle.io/v3/rke_types.go]: nit: This should probably be `SANs` not `sans` as SAN is an acronym.

Comment (HighwayofLife) [reply apis/management.cattle.io/v3/rke_types.go]: Which part? -- Can't be the yaml, as we have nothing else that has any capitalization in cluster.yml. Maybe the variable?

Comment (ibuildthecloud) [reply apis/management.cattle.io/v3/rke_types.go]: Sorry, yes the variable.  Just to follow go naming conventions.  Go capitalizes acronyms.

**Comment by deniseschannon** [comment]
@HighwayofLife could you fix based on @ibuildthecloud comments?

**Comment by HighwayofLife** [comment]
@deniseschannon done, go-generated and rebased. -- waiting for CI build to complete now.

**Pull request merged**